src/tools/sss_override.c

```diff
@@ -81,14 +82,20 @@ static errno_t parse_cmdline(struct sss_cmdline *cmdline,
                               &orig_name, &domain);
     if (ret != EOK) {
         ERROR("Unable to parse name %s.\n", input_name);
-        return ret;
+        goto done;
     }
 
-    *_input_name = input_name;
+    *_input_name = strdup(input_name);
```

Comment:
Shouldn't we check whether `NULL` was returned?

Reply:
You are right